larray_editor/api.py

```diff
@@ -10,7 +12,7 @@
 
 from larray_editor.editor import REOPEN_LAST_FILE, MappingEditor, ArrayEditor
 
-__all__ = ['view', 'edit', 'compare', 'REOPEN_LAST_FILE']
+__all__ = ['view', 'edit', 'compare', 'REOPEN_LAST_FILE', 'run_editor_on_exception']
```

Comment:
I wonder if we should not do the same as in the larray project: stop using `__all__` and specify all available functions in the main `__init__.py` module?

Reply:
What's the advantage of doing this? In any case, if you want to do this, do it. Both are fine with me.

Reply:
In this case, there only one module using __all__. There is no need to centralize the public api in the main `__init__.py` module. So, we can keep it.